tests/test_jump_to_definition_plugin.py

```diff
@@ -1,11 +1,19 @@
 # TODO: create much more tests for langserver
+import platform
 import time
 
+import pytest
 from sansio_lsp_client import ClientState
 
 from porcupine import get_main_window
 from porcupine.plugins.langserver import langservers
 
+pytestmark = pytest.mark.xfail(
+    platform.python_version_tuple()[:2] == ("3", "11"),
+    run=False,
```

Comment:
- I would prefer `sys.version_info >= (3, 11)`, just how I'm used to seeing version checks written.
- Would be IMO better to run the test anyway, so `strict=True` instead of `run=False`. This way I won't forget to update this when I fix the Python 3.11 situation.

Reply:
> I would prefer...

Will do, believe it or not I've never actually had to do that before so I just did what seemed to work.

> better to run the test anyway...

Will do, makes sense.

Reply:
802e38b fixes